tests/test_nexus_client.py

```diff
@@ -1,9 +1,11 @@
 # -*- coding: utf-8 -*-
 import pytest
+import requests
 
 import nexuscli
 from nexuscli import exception
-from nexuscli.nexus_client import NexusClient
+from nexuscli.nexus_client import NexusClient, NexusConfig
```

Comment:
`NexusConfig` is in `nexus_config` IIRC.

Reply:
You're right, I've fixed it